.github/ISSUE_TEMPLATE/submission.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Proposal submission
+about: Propose a session for Our Networks 2019.
+title: '[Submission Title]'
+labels: ''
+assignees: ''
+---
```

Comment:
We didn't have frontmatter in the 2018 template. Are we intending to use the `name` and `about` fields in some way? Looking at the GH docs, `title`, `labels`, and `assignees` will pre-populate the fields.

Reply:
We have to use these because GitHub implemented a new Issue Template workflow, that is my accidental testing in #2. I think this will pre-populate labels and assignees as blank if we include in this way, as we are supposed to. We previously included the submission title as a field to change in the template which I have just moved to the Title (so issue title)

Reply:
Reconfirmed this works as expected, @garrying you can test here: https://github.com/dcwalk/test-issue-templates/issues/new/choose

Reply:
Ohh, got it. The template flow is new to me.

Reply:
Me too! I think this is a v.recent shift